change host file entry for Mac (#39760)
kubernetes/website: content/en/docs/tasks/access-application-cluster/ingress-minikube.md

   If you are running Minikube locally, use `minikube ip` to get the external IP.
   The IP address displayed within the ingress list will be the internal IP.
   {{< /note >}}
   
   {{< note >}}

>    For Mac, if you are running Minikube locally with Docker, point to localhost in the hosts file instead of the minikube ip

Reviewer: ```suggestion
   For Mac, if you are running Minikube locally with Docker Desktop, point to the loopback IP address
   in the hosts file instead of the minikube IP address. For example:
```

   
    ```none
    127.0.0.1 hello-world.info
    ```

>    And then use `minikube tunnel`

Reviewer: ```suggestion
   and then run `minikube tunnel` (you may need to provide a superuser password). Leave that
   running whilst you continue the task.
```

   If you are running Minikube locally, use `minikube ip` to get the external IP.
   The IP address displayed within the ingress list will be the internal IP.

>    {{< /note >}}
>    
>    {{< note >}}

Reviewer: ```suggestion

```
